hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1453,43 +1453,43 @@ public String getHbaseTableName() {
   }
 
   public int getHbaseIndexGetBatchSize() {
-    return getInt(HoodieHBaseIndexConfig.GET_BATCH_SIZE);
+    return getIntOrDefault(HoodieHBaseIndexConfig.GET_BATCH_SIZE);
```

Comment:
Thanks for reporting the issue.  Actually, all of these changes are not needed if the `HoodieHBaseIndexConfig` defaults are set properly.  It looks like that the root cause is that the builder of `HoodieWriteConfig` does not set defaults for `HoodieHBaseIndexConfig` in `setDefaults()`, like other Hudi config class.  Could you fix that and revert all getter changes instead?

Reply:
Thanks for your comments, I updated the pr.